src/Fabulous.XamarinForms/Attributes.fs

```diff
@@ -24,6 +25,59 @@ type ValueEventData<'data, 'eventArgs> =
 module ValueEventData =
     let create (value: 'data) (event: 'eventArgs -> obj) = { Value = value; Event = event }
 
+/// Xamarin Forms specific attributes that can be encoded as 8 bytes
+module SmallScalars =
+    /// In reality XF takes 7 values of 4 bytes each.
+    /// Which is larger than 8 bytes an unusual to represent a color
+    /// So we are just going to bluntly convert it into 8 bytes in rgba form
+    /// note that we allocate 2 bytes for each channel in RGBA format to improve precision
+    /// That being said, it is a lossy conversion
+    module Color =
```

Comment:
@TimLariviere this one I'm not sure about, I think it is worth it but the color value might not be exactly the same, that said it is going to be less than 0.001% accuracy loss, thus I think it is fine

Reply:
I think this is fine too. Most apps are usually "standard" colors anyway, where the accuracy loss won't be an issue.
The potential perf boost will surely be way more beneficial.

If some people really really need that complete accuracy, it will still be easy enough to declare another attribute using the obj casting.

Reply:
Also would be good to point out that accuracy loss in the XML comment of the color modifiers

Reply:
Added a comment